src/clients/c++/library/CMakeLists.txt

```diff
@@ -135,16 +138,52 @@ if(${TRITON_ENABLE_GRPC})
       ARCHIVE DESTINATION lib
   )
 
+  install(
+      FILES
+      ${CMAKE_CURRENT_BINARY_DIR}/../../../../../grpc/lib/libgpr.a
+      ${CMAKE_CURRENT_BINARY_DIR}/../../../../../grpc/lib/libgrpc.a
+      ${CMAKE_CURRENT_BINARY_DIR}/../../../../../grpc/lib/libgrpc++.a
+      ${CMAKE_CURRENT_BINARY_DIR}/../../../../../grpc/lib/libaddress_sorting.a
+      ${CMAKE_CURRENT_BINARY_DIR}/../../../../../protobuf/lib/libprotobuf.a
+      DESTINATION lib
+  )
+
+  if (NOT WIN32)
+     install(
+         FILES
+         ${CMAKE_CURRENT_BINARY_DIR}/../../../../../c-ares/lib/libcares.a
```

Comment:
My bad. I should have been mentioned that this needs to be done for the other libraries as well.